src/overlays/actors/ovl_En_Kusa/z_en_kusa.c

```diff
@@ -347,131 +350,135 @@ void EnKusa_SetupLiftedUp(EnKusa* this) {
 void EnKusa_LiftedUp(EnKusa* this, GlobalContext* globalCtx) {
     if (Actor_HasNoParent(&this->actor, globalCtx)) {
         this->actor.room = globalCtx->roomCtx.curRoom.num;
-        func_80A9BBB0(this);
+        EnKusa_SetupFall(this);
         this->actor.velocity.x = this->actor.speedXZ * Math_SinS(this->actor.world.rot.y);
         this->actor.velocity.z = this->actor.speedXZ * Math_CosS(this->actor.world.rot.y);
         this->actor.colChkInfo.mass = 240;
         this->actor.gravity = -0.1f;
-        EnKusa_Fall(this);
-        func_80A9B174(&this->actor.velocity, 0.005f);
+        EnKusa_UpdateVelY(this);
+        EnKusa_RandScaleVecToZero(&this->actor.velocity, 0.005f);
         func_8002D7EC(&this->actor);
         Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 7.5f, 35.0f, 0.0f, 0xC5);
         this->actor.gravity = -3.2f;
     }
 }
 
-void func_80A9BBB0(EnKusa* this) {
-    EnKusa_SetupAction(this, func_80A9BC1C);
-    D_80A9C1D0 = -0xBB8;
-    D_80A9C1D8 = ((Rand_ZeroOne() - 0.5f) * 1600.0f);
-    D_80A9C1D4 = 0;
-    D_80A9C1DC = 0;
+void EnKusa_SetupFall(EnKusa* this) {
+    EnKusa_SetupAction(this, EnKusa_Fall);
+    rotSpeedXtarget = -0xBB8;
+    rotSpeedYtarget = (Rand_ZeroOne() - 0.5f) * 1600.0f;
+    rotSpeedX = 0;
+    rotSpeedY = 0;
 }
 
-void func_80A9BC1C(EnKusa* this, GlobalContext* globalCtx) {
+void EnKusa_Fall(EnKusa* this, GlobalContext* globalCtx) {
     s32 pad;
     Vec3f contactPos;
 
-    if (this->actor.bgCheckFlags & 11) {
-        if (!(this->actor.bgCheckFlags & 32)) {
+    if (this->actor.bgCheckFlags & (1 | 2 | 8)) {
+        if (!(this->actor.bgCheckFlags & 0x20)) {
```

Comment:
would prefer if it was just left as hex in all cases tbh, to make for easier search and replacing when we make defines